usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/TestFrameworkAssertions.java

```diff
@@ -39,122 +35,179 @@
  * @author m4rkmckenna on 11/11/2015.
  */
 public class TestFrameworkAssertions {
-    private static final Logger LOG = LoggerFactory.getLogger(TestFrameworkAssertions.class);
+
+    public static final String IS_NULL = "isNull";
+    public static final String NOT_NULL = "notNull";
+    public static final String IS_EQUAL_TO = "isEqualTo";
+    public static final String EQUAL_TO = "equalTo";
+    public static final String EQUALS = "equals";
+    public static final String MATCHES = "matches";
```

Comment:
in time (not now) i'd like to introduce aliases across brooklyn, and then allow `matches_regex` and `regex` here.  `matches` is too vague, is it literal, is it glob, is it regex.  but okay for now (and i see you've not changed that in this PR anyway).